apps/dashboard/test/fixtures/sys_with_gateway_apps/bc_jupyter/form.yml

```diff
@@ -127,34 +127,34 @@ attributes:
           "2.7",
           data-option-for-node-type-advanced: false,
           data-option-for-node-type-other-40ish-option: false,
-          data-set-bc-account-other: 'other_account_python27',
-          data-set-bc-account: 'python27'
+          data-suggest-bc-account-other: 'other_account_python27',
+          data-suggest-bc-account: 'python27'
```

Comment:
You changed all these `data-set` to `data-suggest`. This implies to me that there's some compatibility issue with `data-set`, is there?

Reply:
`data-set-` was behaving how `data-suggest-` behaves  now in some tests. So `data-set-` was not disabling the set select option before. This does make it a breaking change.

Reply:
I guess I'll have to look over the implementation, but I don't think we should break `data-set` directives.  We definitely can't in 4.0, but I wouldn't want to in future versions either really.